reV/supply_curve/points.py

```diff
@@ -466,6 +467,37 @@ def exclusion_weighted_mean(self, arr, drop_nan=True):
 
         return mean
 
+    def mean_wind_dirs(self, arr):
+        """
+        Calc the mean wind directions at every time-step
+
+        Parameters
+        ----------
+        arr : np.ndarray
+            Array of wind direction data.
+        Returns
+        -------
+        mean_wind_dirs : np.ndarray | float
+            Mean wind direction of arr masked by the binary exclusions
+        """
+        excl = self.excl_data_flat[self.bool_mask]
+        gids = self._gids[self.bool_mask]
+        if len(arr.shape) == 2:
+            arr_slice = (slice(None), gids)
+            ax = 1
+
+        else:
+            arr_slice = gids
+            ax = 0
+
+        angle = np.radians(arr[arr_slice], dtype=np.float32)
+        sin = np.mean(np.sin(angle) * excl, axis=ax)
+        cos = np.mean(np.cos(angle) * excl, axis=ax)
+
+        mean_wind_dirs = np.degrees(np.arctan2(sin, cos))
```

Comment:
only comment is that it looks like this set of functions will give us degrees in range -180 to 180. Is this what we want?

Reply:
Great catch! No we want 0-359. Let me check the WTK code as I did this there...

Reply:
@grantbuster this should be fixed, my checking my math?

Reply:
Yeah looks good!